plugins/modules/zos_archive.py

```diff
@@ -373,7 +381,7 @@
       - C(archive) when the file is an archive.
       - C(compress) when the file is compressed, but not an archive.
       - C(incomplete) when the file is an archive, but some files under
-        I(path) were not found.
+        I(source) were not found.
```

Comment:
I think this should be I(src), not I(source).... you changed other references above to use the shorter name

Reply:
You are right Rich, will update this